etcdctl/ctlv3/command/printer_json.go

```diff
@@ -47,6 +79,7 @@ func (p *jsonPrinter) MemberList(r clientv3.MemberListResponse) {
 		printJSON(r)
 	}
 }
+func (p *jsonPrinter) MemberAdd(r clientv3.MemberAddResponse) { p.printJSON(r) }
```

Comment:
I think we can construct a fake `MemberAddResponse` object, and call this method directly to ensure that it generates the expected output with and without `--hex`.

We also need to make similar change for some other commands. So let's make this PR as a good example for following PRs to follow.